doc/source/ray-observability/ray-distributed-debugger.rst

```diff
@@ -77,6 +77,50 @@ Start a Ray cluster
     After checking that `ssh -p 2222 ray@localhost` works, set up VS Code as described in the
     `VS Code SSH documentation <https://code.visualstudio.com/docs/remote/ssh>`_.
 
+  .. tab-item:: KubeRay (Code Server)
+
+    Follow the instructions in :doc:`the RayCluster quickstart <../cluster/kubernetes/getting-started/raycluster-quick-start>` to set up a cluster.
+    
+    A simpler approach is to run a browser-based VS Code (Code Server) as a sidecar container in the Ray head pod. This eliminates network connectivity issues by placing VS Code inside the Kubernetes cluster.
+
+    Add a sidecar container to the Ray head pod and configure a shared volume. Modify your Ray head pod template with the following additions:
+    
+    .. code-block:: yaml
+
+        # In your RayCluster YAML, under spec.headGroupSpec.template.spec
+        containers:
+          - name: ray-head
+            # ... your existing ray-head configuration ...
+            # Add this volumeMount:
+            volumeMounts:
+              - mountPath: /tmp/ray
+                name: shared-ray-volume
+          # Add this sidecar container:
+          - name: vscode-debugger
+            image: docker.io/onesizefitsquorum/code-server-with-ray-distributed-debugger:4.101.2
+            ports:
+              - containerPort: 8443
+            volumeMounts:
+              - mountPath: /tmp/ray
+                name: shared-ray-volume
+            env:
+              - name: DEFAULT_WORKSPACE
```

Comment:
Could you add some comment on what this does? Does it tell vscode where the code lives? Is it needed?

Reply:
Good idea！Added！